brainscore/metrics/ceiling.py

```diff
@@ -160,3 +162,42 @@ def _ceiling(self, identifier):
     'cons': InternalConsistency,
     None: NoCeiling,
 }
+
+
+class SplitHalvesConsistencyBaker:
```

Comment:
move this inside the benchmark definition; the `ceiling.py` file is really just meant to be generic utils, whereas this is defining a benchmark-specific estimation